### DISCUSSION TITLE
Related to password

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3194] [Q&A]
Hi.

1) Is there any API available to fetch the user password in Nocodb? 

2) I tried validating email using  
/../api/v1/auth/email/validate/sumathyv@lotusinterworks.com 

but getting   response as "msg": "Invalid verification url".
can you please provide an example

Thanks and Regards,
Sumathy Venugopal

**Comment by wingkwong** [comment]
1. No. What's the use case of fetching user password?
2. You should pass the token instead. See [here](https://all-apis.nocodb.com/#tag/Auth/operation/auth-email-validate)

**Reply by MGW-SumathyV** [reply]
1. I wanted to open the Nocodb GUI (project list page) from my application without asking user to signin.

**Reply by wingkwong** [reply]
Do you mean embedding project list page in your application? What would happen if users click the project list item? Would it be redirecting to NocoDB GUI?

**Reply by MGW-SumathyV** [reply]
Work flow would be:
user logs into my application
when they click on nocodb link user will be auto sign in to Nocodb and user will be redirected to the project list page

**Reply by wingkwong** [reply]
Currently it's not supported.

**Reply by MGW-SumathyV** [reply]
> Do you mean embedding project list page in your application? What would happen if users click the project list item? Would it be redirecting to NocoDB GUI?

yes thats right. Is this supported currently? could you please confirm
